src/main/java/com/epam/aidial/core/service/PublicationService.java

```diff
@@ -612,24 +611,21 @@ private void deletePublicResources(List<Publication.Resource> resources) {
 
     private boolean checkResource(ResourceDescription descriptor) {
         return switch (descriptor.getType()) {
```

Comment:
simplify and inline method: resources.hasResource(descriptor)

Reply:
Done.